### ISSUE TITLE
130.0.1 (64-Bit)

### ISSUE THREAD

**Issue by pitgas** [xpmn/firefox-to-deepl#36]
No longer works. The input field in the popup or new tab is empty, and the text is not transferred.
Fresh Firefox installation, no other add-ons installed.
Win 10; Fire Fox 130.0.1 (64-Bit)

**Comment by ltstereo** [comment]
same here Win 11 Fox 130.0.1 (64-Bit)

**Comment by zznidar** [comment]
The problem appears to have been caused by DeepL changing the URL for translation.

Previously, passing `#null/en` caused it to auto-detect source language and translate to English. With the change, however, `#null` is not a valid language and causes the input to remain empty.

**Comment by xpmn** [comment]
I don't know what to put instead of null, tried some other variants, but it doesn't help

**Comment by no-one** [comment]
@xpmn This seems to be the language of the word that should be translated:
``https://www.deepl.com/de/translator#en/de/hello``
``https://www.deepl.com/de/translator#de/es/hallo``

**Comment by xpmn** [comment]
@no-one yes, I know, but previously I passed null to initiate autodetect. Now it stop working, so I've set to en just to make it at least not lost text

**Comment by nickzam** [comment]
Thank you for the prompt fix. Could you please upload update to Mozilla ?

**Comment by xpmn** [comment]
It's on review already 
![image](https://github.com/user-attachments/assets/dcbda023-e4dc-4044-91ed-dbcb0e3110fe)

**Comment by xpmn** [comment]
yes it was ¯\_(ツ)_/¯
but whole this extension is a hack

**Comment by nickzam** [comment]
FYI, I was tired of waiting for Firefox to review and installed it directly from github. It was much easier that I expected, cheers for the fix :)

**Comment by zznidar** [comment]
I am unable to check right now – but does language detection occur client-side or server-side? Because in the latter case, we could simply send a request to DeepL server, get the language in return, and only then open the URL with the correct languages.
